.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - STACK=heroku-18 IMAGE_TAG=heroku/heroku:18 PRIVATE_IMAGE_TAG=heroku/heroku-private:18
   - STACK=heroku-20 IMAGE_TAG=heroku/heroku:20 PRIVATE_IMAGE_TAG=heroku/heroku-private:20
 script:
+- find . -type f \( -name "*.sh" -o -path "*/bin/*" \) ! -name '*.jq' | xargs -t shellcheck
```

Comment:
Can't use only bash globbing (eg `shellcheck **/bin/* **/*.sh`) since otherwise shellcheck will be passed the same filename twice, and currently the input file list is not de-duplicated, so any failures would show up in the results twice:
https://github.com/koalaman/shellcheck/issues/1930